physionet-django/export/views.py

```diff
@@ -61,3 +64,45 @@ def get(self, request, project_slug, version, *args, **kwargs):
         project = get_object_or_404(PublishedProject, slug=project_slug, version=version)
         serializer = PublishedProjectDetailSerializer(project)
         return Response(serializer.data)
+
+
+class PublishedProjectSearch(mixins.ListModelMixin, generics.GenericAPIView):
+    """
+    Search for a Published Project using the get_content function inside Search Module's views.py
+    """
+
+    serializer_class = PublishedProjectSerializer
+
+    def get_queryset(self):
+        resource_type = self.request.POST.get('resource_type', '0,1,2,3') or self.request.GET.get('resource_type',
```

Comment:
change to string and keep any integer logic internal

Reply:
This has been updated to take in the string values of the resource types, namely database, software, challenge, model, or all.